lib/global_registry/base.rb

```diff
@@ -23,6 +24,24 @@ def self.get(params = {}, headers = {})
     def get(params = {}, headers = {})
       request(:get, params, nil, headers)
     end
+    def self.get_all_pages(params = {}, headers = {})
```

Comment:
We moved to instance methods because we want different clients to be able to point at different GR urls or use different access tokens. That was a requirement for MAPI. You won't need to have different tokens for the syncing stuff, but it will still be helpful. Plus, if you made it instance level, then it would only instantiate one GR object instead of one for every page since `Base.get` makes a new instance internally.